actions/views/root.js

```diff
@@ -57,11 +57,11 @@ export function loadTranslations(locale, url) {
         // No need to go to the server for EN
         if (locale === 'en') {
             copyAndDispatchTranslations(dispatch, translations, en, locale);
-            return;
         }
         Client4.getTranslations(url).then((serverTranslations) => {
```

Comment:
i was hoping we'd put this in the `else` block and turn this call into async/await and call `copyAndDispatchTranslations`. The way you have it right now this will always be called, which I don't think is what we want here.

And since `copyAndDispatchTranslations` will only be called once maybe remove the function `copyAndDispatchTranslations` and dispatch the action directly at the end of this function.

Reply:
I'm quite new to js, I'll see what i can do.